linode/manager
# feat: [M3-8676] - Provide Customer Notice for OS Distro Nearing EOL/EOS

## Description 📝
Add Notice for OS Distro Nearing EOL/EOS

## Changes  🔄
- Added a notice for images that are nearing EOL/EOS

## Target release date 🗓️
10/12

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/9a8c27c3-fb4f-43c2-a913-6bf217e916b6) | ![image](https://github.com/user-attachments/assets/31e95350-bf3b-476d-83fe-bd55ece67946) |
| ![image](https://github.com/user-attachments/assets/175f7880-4e42-4bf2-b670-fae458e32b7f) | ![image](https://github.com/user-attachments/assets/4d08e0ab-450a-4fe0-aec0-21186a4528c9)|

## How to test 🧪

### Verification steps

-  Navigate to the Create Linode page
- Choose an OS that is deprecated
- You should see a notice similar to the one in `Preview`

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅
